src/components/Comments/Comment.js

```diff
@@ -377,7 +377,7 @@ const styles = StyleSheet.create({
   time: {
     fontFamily: fonts.inter[400],
     fontSize: 10,
-    color: '#828282',
+    color: COLORS.blackgrey,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hard-coded color value in the `time` style has been replaced with a dynamic color from the theme object. This improves consistency and maintainability of the code.
```diff
-    color: '#828282',
+    color: COLORS.blackgrey,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->